### DISCUSSION TITLE
Always preserve scrolling in Link

### DISCUSSION THREAD

**Discussion by scarm01** [inertiajs/inertia#1200] [Help]
Hi, is there a way to always preserve scrolling by default when visiting links without having to use `preserve-scroll` every time?

**Comment by RobertBoes** [comment]
That should be possible with #1052, it's just not documented yet

**Comment by scarm01** [comment]
I tried adding inside createInertiaApp the following piece of code but it doesn't work:
```
createInertiaApp({
  ...
  visitOptions(options, url) {
          options.preserveScroll = true
          return options
      }
  ...
})
```
